### ISSUE TITLE
False-positive message "Avoid undefined variables" on short list syntax

### ISSUE THREAD

**Issue by eremin** [phpmd/phpmd#725]
- PHPMD version: 2.8.1
- PHP Version: 7.2.24
- Installation type: composer
- Operating System / Distribution & Version: Kubuntu 19.04

### Current Behavior
False-positive message "Avoid using undefined variables" on short list assignment
```php
[$abc, $def] = [1, 2];
```

### Expected Behavior
Recognize assignment via shot list syntax

### Steps To Reproduce:
sample code is above

**Comment by kylekatarnls** [comment]
This should be fixed with #770 and you can yet test it requiring "phpmd/phpmd": "dev-master"
